src/app/services/llm/claude.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from datetime import datetime
+from enum import Enum
+from typing import Dict, Generator, List, Union
+from anthropic import Client
+from .utils import CHAT_PROMPT
+
+class ClaudeModel(str, Enum):
+    Opus: str = "claude-3-opus-20240229"
+    Sonnet: str = "claude-3-sonnet-20240229"
+    Haiku: str = "claude-3-haiku-20240307"
```

Comment:
For the sake of conventions, could you use UPPER_CASE for the class attributes of enums please?